manifests/vsphere-67u3/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -1,3 +1,5 @@
+# Minimum Kubernetes version - 1.16
+# For prior releases make sure to add required --feature-gates flags for side car containers
```

Comment:
Remove "for side car containers" as most feature gates are for api server, kube controller manager, kubelet, etc. 

Same below.

Reply:
done